src/controller/controller.ts

```diff
@@ -471,11 +471,14 @@ class Controller extends events.EventEmitter {
         debug.log(`Device leave '${payload.ieeeAddr}'`);
 
         const device = Device.byIeeeAddr(payload.ieeeAddr);
-        if (device) {
-            debug.log(`Removing device from database '${payload.ieeeAddr}'`);
-            device.removeFromDatabase();
+        if (!device) {
+            debug.log(`Device leave is from unknown device '${payload.ieeeAddr}'`);
```

Comment:
`Device leave is from unknown or already deleted device`
I think would that be more precise for debug purposes.

Reply:
done!